src/sentry/lang/native/sources.py

```diff
@@ -342,7 +387,7 @@ def validate_sources(sources, schema=SOURCES_SCHEMA):
         ids.add(source["id"])
 
 
-def parse_sources(config):
+def parse_sources(config, filter_appconnect):
```

Comment:
I decided to not give `filter_appconnect` a default value as before because I find it hard to keep track of what the default behavior is at each callsite.